### PR TITLE
Remove line breaks from selector string.

### PR DESCRIPTION
**Pull request by montehurd**

### CODE REVIEW

File: src/transform/FooterContainer.js [hunk 1]
```diff
@@ -22,11 +22,12 @@ const updateBottomPaddingToAllowReadMoreToScrollToTop = window => {
  * @return {void}
  */
 const updateLeftAndRightMargin = (margin, document) => {
-  const elements = Polyfill.querySelectorAll(document, `
-    #pagelib_footer_container_menu_heading,
-    #pagelib_footer_container_readmore,
-    #pagelib_footer_container_legal
-  `)
+  const selectors = [
```
Comment (niedzielski) [inline src/transform/FooterContainer.js]: Is this fixing some weird platform bug or is it a purely nonfunctional clean-up change / chore?

Comment (montehurd) [inline src/transform/FooterContainer.js]: In the js file produced by babel I noticed it had '\n' inside of this string and it just seemed odd. Didn't appear to affect functionality, but thought it was easy to remove those.

Comment (niedzielski) [inline src/transform/FooterContainer.js]: Find with me. Just wanted to double check I understood the change!

Comment (montehurd) [inline src/transform/FooterContainer.js]: Sorry I should have had a better description! :)

**Pull request merged**